zone/bonuses.cpp

```diff
@@ -1568,6 +1574,25 @@ void Mob::ApplyAABonuses(const AA::Rank &rank, StatBonuses *newbon)
 			newbon->Pet_Add_Atk += base1;
 			break;
 
+		case SE_Weapon_Stance: 
+		{
+			if (IsValidSpell(base1)) { //base1 is the spell_id of buff
```

Comment:
Whenever there is a comment explaining things, there might be an opportunity to codeify this

Eg `int32 spell_id = base1;`